### PR TITLE
Short term storage loading ini files

### PR DESCRIPTION
**Pull request by payetvin**
branched from #1163

### CODE REVIEW

File: src/libs/antares/study/parts/short-term-storage/container.cpp [hunk 1]
```diff
@@ -25,23 +25,54 @@
...
-    return false;
+    const std::string pathIni = path +  SEP + "list.ini";
+
+    IniFile ini;
+    if (!ini.open(path))
+        return false;
+
+    for (auto* section = ini.firstSection; section; section = section->next)
+    {
+        if (!ContainerLoadFromSection(pathIni, *section))
```
Comment (flomnes) [inline src/libs/antares/study/parts/short-term-storage/container.cpp:54]: This method is missing ?

File: src/libs/antares/study/parts/short-term-storage/container.cpp [hunk 2]
```diff
@@ -25,23 +25,54 @@
...
+
+bool Container::loadFromSection(const std::string& pathIni, const IniFile::Section& section)
+{
+    if (section.name.empty())
+        return false;
+
+    Unit unit;
+    unit.loadFromFolder(pathIni);
+    storages.insert({ section.name, unit });
+    return true;
```
Comment (flomnes) [inline src/libs/antares/study/parts/short-term-storage/container.cpp:74]: Multiple units may be present, so a loop of some kind will be required here

File: src/libs/antares/study/parts/short-term-storage/container.h [hunk 3]
```diff
@@ -28,6 +28,9 @@
 #pragma once
 #include <string>
 #include <map>
+
+#include <antares/study.h>
+
```
Comment (flomnes) [inline src/libs/antares/study/parts/short-term-storage/container.h:33]: Why do you need this header ?

**Comment by payetvin** [comment]
Bad start, switched to a new branch and PR

**Pull request closed without merging**